src/main/java/TaskList.java

```diff
@@ -0,0 +1,26 @@
+import java.util.ArrayList;
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTask(String userInput) throws HandleException {
+        // Similar to addTask method in TaskManager, but refactored for this class
```

Comment:
Make sure the comments are for the reader and not private notes for yourself. It should help in increasing code readability.